tools/wheels/cibw_before_build.sh

```diff
@@ -22,30 +22,40 @@ fi
 if [[ $(python -c"import sys; print(sys.maxsize)") < $(python -c"import sys; print(2**33)") ]]; then
     echo "No BLAS used for 32-bit wheels"
     export INSTALL_OPENBLAS=false
-elif [[ $(python -c"import sysconfig; print(sysconfig.get_platform())") == "win-arm64" ]]; then
-    echo "No BLAS used for ARM64 wheels"
-    export INSTALL_OPENBLAS=false
 elif [ -z $INSTALL_OPENBLAS ]; then
     # the macos_arm64 build might not set this variable
     export INSTALL_OPENBLAS=true
 fi
 
 # Install Openblas from scipy-openblas64
 if [[ "$INSTALL_OPENBLAS" = "true" ]] ; then
-    echo PKG_CONFIG_PATH $PKG_CONFIG_PATH
+    # by default, use scipy-openblas64
+    OPENBLAS=openblas64
+    # Possible values for RUNNER_ARCH in github are
+    # X86, X64, ARM, or ARM64
+    # TODO: should we detect a missing RUNNER_ARCH and use platform.machine()
+    #    when wheel build is run outside github?
+    # On 32-bit platforms, use scipy_openblas32
+    # On win-arm64 use scipy_openblas32
+    if [[ $RUNNER_ARCH == "X86" || $RUNNER_ARCH == "ARM" ]] ; then
```

Comment:
The X86 part does nothing here, since `INSTALL_OPENBLAS` is false for 32-bit platforms.

Reply:
Okay sure, resolving.